db/index.js

```diff
@@ -5,7 +5,7 @@ const Promise = require('bluebird');
 mongoose.Promise = Promise;
 
 // The default url only works on local machines
-var connectionURL = 'mongodb://localhost/greenfield';
+let connectionURL = 'mongodb://localhost/greenfield';
```

Comment:
As the reviewer, it would be great if there were some comments to explain why you went back and forth with the let vs var for this line.